src/main/java/com/github/danielflower/mavenplugins/gitlog/GenerateMojo.java

```diff
@@ -64,7 +64,7 @@
 	 * If true, then an Asciidoc changelog will be generated.
 	 */
 	@Parameter(defaultValue = "false")
-	private boolean generatAsciidocChangeLog;
+	private boolean generateAsciidocChangeLog;

```

Comment:
This is a breaking change... so the misspelled property needs to remain there, but should be marked as deprecated, and should be used if it is true. Or you could revert this change.